transform/models/marts/fact_commits.sql

```diff
@@ -2,6 +2,7 @@ select
     strftime('%m/%d/%Y', event_date) as date,
     repo_id,
     count(*) as count 
-from {{ ref("stg_gharchive") }}
-where event_type = 'Push'
+from (
+    {{ fact_event_type('Push') }}
```

Comment:
We could simplify this by making the whole file only a call to the macro:

```sql+jinja
{{ fact_event_type('Push') }}
```

Even if we wanted to keep the strftime conversion, we can just move that common logic to the macro code by making the macro:

```sql+jinja
{% macro fact_event_type(event_type) -%} 
  select 
    stftime('...', event_date) as date, 
    repo_id, 
    user 
  from {{ ref('stg_gharchive') }} 
  where event_type = '{{ event_type }}' 
{%- endmacro %} 
```

We could even add the count(*) although we won't need it here as each row is a single event.